scripts/frontmatter/get-frontmatter.js

```diff
@@ -0,0 +1,93 @@
+const frontmatter = require('@github-docs/frontmatter');
+const { TYPES } = require('../constants');
+
+const GATSBY_CONTENT_TYPES = {
+  [TYPES.BASIC_PAGE]: 'page',
+  [TYPES.API_DOC]: 'apiDoc',
+  [TYPES.RELEASE_NOTE]: 'releaseNote',
+  [TYPES.RELEASE_NOTE_PLATFORM]: 'releaseNotePlatform',
+  [TYPES.TROUBLESHOOTING]: 'troubleshootingDoc',
+  [TYPES.WHATS_NEW]: 'nr1Announcement',
+};
+
+const GATSBY_TEMPLATE = {
+  [TYPES.BASIC_PAGE]: 'basicDoc',
+  [TYPES.API_DOC]: 'basicDoc',
+  [TYPES.RELEASE_NOTE]: 'basicDoc',
+  [TYPES.RELEASE_NOTE_PLATFORM]: 'basicDoc',
+  [TYPES.TROUBLESHOOTING]: 'basicDoc',
+  [TYPES.WHATS_NEW]: 'basicDoc',
+};
+
+const FRONTMATTER_TEMPLATE = {
+  [TYPES.BASIC_PAGE]: 'page',
+  [TYPES.API_DOC]: 'apiDoc',
+  [TYPES.RELEASE_NOTE]: 'releaseNote',
+  [TYPES.RELEASE_NOTE_PLATFORM]: 'releaseNotePlatform',
+  [TYPES.TROUBLESHOOTING]: 'troubleshooting',
+  [TYPES.WHATS_NEW]: 'whatsNew',
+};
+
+const getFrontmatter = (type, doc) => {
+  const defaultFrontmatter = {
+    title: doc.title.replace(':', '-'),
+    contentType: GATSBY_CONTENT_TYPES[type],
```

Comment:
Can you remind me again why we want to differentiate the content type from the template? What are we going to use this value for?

Reply:
I believe content types are used as a meta tag for SwiftType.

Reply:
yeah, swiftype is the main thing that we use content types for. we filter out category/index pages for better results and the search page on docs allows a user to filter for a specific content type

Reply:
Thats right! Thanks 😄. I knew I was forgetting something important